3.4/Dockerfile

```diff
@@ -0,0 +1,97 @@
+# vim:set ft=dockerfile:
+
+# Do not edit individual Dockerfiles manually. Instead, please make changes to the Dockerfile.template, which will be used by the build script to generate Dockerfiles.
+
+# By policy, the base image tag should be a quarterly tag unless there's a
+# specific reason to use a different one. This means January, April, July, or
+# October.
+
+FROM cimg/base:2024.12
```

Comment:
While the 3.3 image [1] has not complied the policy "By policy, the base image tag should be a quarterly tag unless there's a specific reason to use a different one. This means January, April, July, or October.", what do you think about use of `cimg/base:2024.10` (or upcoming `cimg/base:2025.01`)?

- 1: https://github.com/CircleCI-Public/cimg-ruby/blob/4e5a5acc7a92dfe919330f98a764131ed52894f2/3.3/Dockerfile#L9